usaspending_api/database_scripts/matview_generator/summary_view_cfda_number.json

```diff
@@ -40,9 +40,12 @@
     }, {
       "name": "type",
       "columns": [
-        {"name": "\"action_date\"", "order": "DESC NULLS LAST"},
         {"name": "\"type\""}
       ]
+    }, {
```

Comment:
Can we add these added `pulled_from` and removed `action_date` checks to the technical notes?